docs/en/setup/backend/log-analyzer.md

```diff
@@ -11,8 +11,29 @@ Java agent provides toolkit for
 to report logs through gRPC with automatic injected trace context.
 
 [SkyWalking Satellite sidecar](https://github.com/apache/skywalking-satellite) is a recommended proxy/side to
-forward logs including to use Kafka MQ to transport logs. When use this, need to open [kafka-fetcher](backend-fetcher.md#kafka-fetcher).
+forward logs including to use Kafka MQ to transport logs. When use this, need to open [kafka-fetcher](backend-fetcher.md#kafka-fetcher)
+and enable configs `enableNativeProtoLog`.
 
+### Log files collector
+
+Java agent provides toolkit for
+[log4j](../service-agent/java-agent/Application-toolkit-log4j-1.x.md#print-skywalking-context-in-your-logs),
+[log4j2](../service-agent/java-agent/Application-toolkit-log4j-2.x.md#print-skywalking-context-in-your-logs),
+[logback](../service-agent/java-agent/Application-toolkit-logback-1.x.md#print-skywalking-context-in-your-logs)
+to report logs through files with automatic injected trace context.
+
+Log framework config examples:
+- [log4j1.x fileAppender](../../../../test/e2e/e2e-service-provider/src/main/resources/log4j.properties)
+- [log4j2.x fileAppender](../../../../test/e2e/e2e-service-provider/src/main/resources/log4j2.xml)
+- [logback fileAppender](../../../../test/e2e/e2e-service-provider/src/main/resources/logback.xml)
+
+You can use [Filebeat](https://www.elastic.co/cn/beats/filebeat) 、[Fluentd](https://fluentd.org) to
+collect file logs including to use Kafka MQ to transport logs. When use this, need to open [kafka-fetcher](backend-fetcher.md#kafka-fetcher)
+and enable configs `enableNativeJsonLog`.
+
+Collector config examples:
+- [filebeat.yml](../../../../test/e2e/e2e-test/docker/kafka/filebeat.yml)
+- [fluentd.conf](../../../../test/e2e/e2e-test/docker/kafka/fluentd.conf)
```

Comment:
We need an update here, https://github.com/apache/skywalking/blob/master/docs/en/protocols/Log-Data-Protocol.md, to update log's JSON format. We don't implement HTTP JSON format receiver yet, right?

Reply:
`We don't implement HTTP JSON format receiver yet, right?`

Yes, need to add and merge into this branch?

Reply:
We don't need to. I mean we should update the doc.